dlt-connector/src/interactions/backendToDb/transaction/TransactionRecipe.role.ts

```diff
@@ -0,0 +1,63 @@
+import { KeyPair } from '@/data/KeyPair'
+import { TransactionBuilder } from '@/data/Transaction.builder'
+import { UserRepository } from '@/data/User.repository'
+import { TransactionBodyBuilder } from '@/data/proto/TransactionBody.builder'
+import { TransactionErrorType } from '@/graphql/enum/TransactionErrorType'
+import { TransactionDraft } from '@/graphql/input/TransactionDraft'
+import { TransactionError } from '@/graphql/model/TransactionError'
+import { sign } from '@/utils/cryptoHelper'
+import { Transaction } from '@entity/Transaction'
+
+export class TransactionRecipeRole {
+  protected transactionBuilder: TransactionBuilder
+
+  public constructor() {
+    this.transactionBuilder = new TransactionBuilder()
+  }
+
+  public async create(transactionDraft: TransactionDraft): Promise<TransactionRecipeRole> {
+    const senderUser = transactionDraft.senderUser
+    const recipientUser = transactionDraft.recipientUser
+
+    // loading signing and recipient account
+    // TODO: look for ways to use only one db call for both
+    const signingAccount = await UserRepository.findAccountByUserIdentifier(senderUser)
```

Comment:
because accountNr is set to default 1 for now, until we have multi accounts in backend and frontend.
What if a User will send gradidos from one of his account to another of his accounts? Then only the transactionNr differ.